### PR TITLE
DM-1261 add Extensible Device Registration reference to connecting devices

### PR DESCRIPTION
**Pull request by dominikschmitz**

### CODE REVIEW

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ If you are subscribed to the required applications you will see other options fo
 
 <img src="/images/users-guide/DeviceManagement/devmgmt-register-devices-custom.png" alt="Register devices">
 
+Microservice developers can also use the [Extensible device registration](https://cumulocity.com/guides/concepts/applications/#extensible-device-registration) and implement a custom registration form that blends seamlessly into the UI.
```
Comment (Oezge-Akin) [inline content/users-guide/device-management-bundle/connecting-devices.md:60]: Shouldn't you add to the develop PR https://github.com/SoftwareAG/c8y-docs/pull/401 then all commits can be grafted to 10.15 together?
And link should be relative path (concepts/applications/#extensible-device-registration) 
Textwise looks good 👍

Comment (sag-eweingaertner) [reply content/users-guide/device-management-bundle/connecting-devices.md:60]: +1 for the text and the idea of moving this to #401. But for the relative paths, I've seen lately that @BeateRixen changed them to absolute URLs. Beate, what is the right way to do it?

Comment (dominikschmitz) [reply content/users-guide/device-management-bundle/connecting-devices.md:60]: I will add this to the other PR. URL -> waiting for feedback.

Comment (dominikschmitz) [reply content/users-guide/device-management-bundle/connecting-devices.md:60]: I will close this PR! Change introduced here: https://github.com/SoftwareAG/c8y-docs/pull/401/commits/2a8968af51c1955dda3eef50c5e7386fa1fbf8ab#diff-7ac3f182cfdd0824689769475d6f7561f087644c17f14ccc658b61100744c203R56
Also moved the question about the URL into the other PR.

**Pull request closed without merging**